Makefile.defs

```diff
@@ -585,13 +576,6 @@ CMAKE_STAGING_DIR=build/staging-$(MACHINE)
 # in that case.
 
 ifdef EMULATE_READER
-	# CMake is difficult about CC and CXX, especially when it's not just a full path
-	CMAKE_C_COMPILER:=$(shell command -v $(strip $(HOSTCC)))
-	CMAKE_CXX_COMPILER:=$(shell command -v $(strip $(HOSTCXX)))
-	ifdef CCACHE
-		CMAKE_C_COMPILER_LAUNCHER:=$(CCACHE)
-		CMAKE_CXX_COMPILER_LAUNCHER:=$(CCACHE)
-	endif
```

Comment:
Same question as above except with greater prejudice (against CMake, that is ^_^).

Reply:
It's not used, not exported, and when cross-compiling, already handled as part of the CMake toolchain definitions.

Reply:
The former I'm not asking about! The latter I'll trust you on.